tests/sources/common/pyconsole_test.py

```diff
@@ -334,3 +334,39 @@ def testError_skipLocked(self):
         %s
         # locked
         """ % hashedAnswer)
+
+
+    def testPassCount_allPassed(self):
+        code="""
+        >>> 2 + 3
+        5
+        >>> 4 + 6
+        10
+        """[1:]
```

Comment:
Re the [1:], I assume it doesn't like the first new line? I'm curious why that isnt a problem in the other test cases above?

Reply:
It creates an empty test case with the new line. That doesn't matter for other scenarios because nothing is checking the total number of test cases or the number of passed tests.